pom.xml

```diff
@@ -88,7 +88,7 @@
     <dependency>
       <groupId>com.cloudbees.jenkins.plugins</groupId>
       <artifactId>kubernetes-credentials-provider</artifactId>
-      <version>0.11</version>
+      <version>0.12.1</version>
```

Comment:
@garethjevans do you actually need this?  this was for a fix to upgrade the fabric8 k8s library and I have no tested that (it has the testing that the OSS user who hit the issue has done)